tests/data/sample_poses.py

```diff
@@ -0,0 +1,90 @@
+"""Sample poses for testing the averaging algorithms.
+
+Authors: Ayush Baid
+"""
+import copy
+from typing import Dict, List, Tuple
+
+import numpy as np
+from gtsam import Pose3, Rot3
+
+
+def generate_relative_from_global(
+    wTi_list: List[Pose3], pose_pairs: List[Tuple[int, int]]
+) -> Dict[Tuple[int, int], Pose3]:
+    """Generate relative poses from global poses.
+
+    Args:
+        wTi_list: global poses.
+        pose_pairs: pairs (i1, i2) to construct relative poses for.
+
+    Returns:
+        Relative pose i2Ti1 for all requested pairs, as a dictionary.
```

Comment:
Please change to: Dict from (i1, i2) to i2Ti1. That makes it clear that the keys are in order (i1, i2).

Reply:
Thanks